Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -403,6 +403,26 @@ def ip_to_indicator_type(ip):
             return None
 
 
+class ScheduleMetadata(object):
+    """
```

Comment:
I think just to avoid confusion , we delete this class. At the beginning I thought this is the class used by CommandResults.
You use this class only in debug-mode code, for clear coding. I think you can replace it with a function or few lines of code in the debug-mode code.

Reply:
ok 👍